internal/interpreter/api.rs

```diff
@@ -1090,28 +1096,39 @@ impl ComponentDefinition {
     pub fn global_properties_and_callbacks(
         &self,
         global_name: &str,
-    ) -> Option<impl Iterator<Item = (String, i_slint_compiler::langtype::Type)> + '_> {
+    ) -> Option<
+        impl Iterator<
+                Item = (
+                    String,
+                    i_slint_compiler::langtype::Type,
+                    i_slint_compiler::object_tree::PropertyVisibility,
+                ),
+            > + '_,
+    > {
         // We create here a 'static guard, because unfortunately the returned type would be restricted to the guard lifetime
         // which is not required, but this is safe because there is only one instance of the unerased type
         let guard = unsafe { generativity::Guard::new(generativity::Id::new()) };
         self.inner
             .unerase(guard)
             .global_properties(global_name)
-            .map(|o| o.map(|(s, t)| (s.to_string(), t)))
+            .map(|o| o.map(|(s, t, v)| (s.to_string(), t, v)))
     }
 
     /// List of publicly declared properties in the exported global singleton specified by its name.
     pub fn global_properties(
         &self,
         global_name: &str,
-    ) -> Option<impl Iterator<Item = (String, ValueType)> + '_> {
+    ) -> Option<
```

Comment:
This is a public function. You can't change the signature. 
Please be careful about public apis.